service/rtc/config.go

```diff
@@ -79,13 +88,31 @@ func (c ICEServerConfig) IsValid() error {
 			return fmt.Errorf("invalid empty URL")
 		}
 
-		if !strings.HasPrefix(u, "stun:") && !strings.HasPrefix(u, "turn:") {
+		if !c.IsSTUN() && !c.IsTURN() {
 			return fmt.Errorf("URL is not a valid STUN/TURN server")
 		}
 	}
 	return nil
 }
 
+func (c ICEServerConfig) IsTURN() bool {
+	for _, u := range c.URLs {
+		if strings.HasPrefix(u, "turn:") {
+			return true
```

Comment:
break early, so only one needs to be valid? (same Q for `IsSTUN` below)

Reply:
I was assuming correctness there really considering it would fail eventually when creating the connection but makes sense to fail as soon as possible especially since these utils are used in `IsValid` as well. Will send a fix, thanks!